README.md

```diff
@@ -122,6 +129,94 @@ Will match
 
 Have fun!
 
+Parameter Interpolation
+-----------------------
+
+Rhumb allows you to take a route and a set of params and produce a path that can be matched against a route.
+
+#### fixed paths
+
+When you give `.interpolate(...)` a route without any declared variables or partial variable parts, then you will not see much changes:
```

Comment:
Maybe this should say that `.interpolate(...)` *will* escape the result if necessary, and show an example?